speech_to_text.py

```diff
@@ -128,41 +140,47 @@ def run_whisper(job):
         "effective_writer_options": writer_options,
         "effective_model_name": model_name,
     }
-    job["extraction_technical_metadata"] = tech_md_dict
 
     return job
 
 
 def upload_results(job):
     """
-    Upload the Whisper output to S3, and put the job file there too.
+    Upload the Whisper output to S3, and put the job file there too. The job
+    file will have the output key added to it, which will contain a list of
+    bucket path names for the results.
     """
     bucket = get_bucket()
 
-    bucket.put_object(Key=f"out/{job['id']}.json", Body=json.dumps(job, indent=2))
-
+    job["output"] = []
     output_dir = get_output_dir(job)
     for path in output_dir.iterdir():
-        # don't upload the media again
-        if path.name in job["media"]:
+
+        # ignore non output files
+        if path.suffix not in [".vtt", ".srt", ".json", ".txt", ".tsv"]:
             continue
 
-        key = f"out/{job['id']}/{path.name}"
+        key = f"{job['id']}/output/{path.name}"
         logging.info(f"wrote whisper result to s3://{bucket.name}/{key}")
         bucket.upload_file(path, key)
+        job["output"].append(key)
+
+    bucket.put_object(Key=f"{job['id']}/job.json", Body=json.dumps(job, indent=2))
+
+    # the files have landed in s3 so the local copies can be deleted so they
+    # don't accumulate in the docker container over time
+    logging.info("deleting local files for job: {output_dir}")
+    shutil.rmtree(output_dir)
```

Comment:
Perhaps output_dir isn't the greatest name for this location since it is also where the input files for Whisper live. https://github.com/sul-dlss/speech-to-text/blob/main/speech_to_text.py#L94-L100